### PR TITLE
[Swift] Add exercise get organized

### PR DESCRIPTION
**Pull request by wneumann**
Adding the Get Organized exercise for the concepts of generic functions and type parameters.

Hopefully everything was done correctly to work with the post-reorg repo.

### CODE REVIEW

File: languages/swift/concepts/generic-functions/about.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+One of the most important concepts in programming is code reuse. One should be able to limit the number of times they write a piece of code in order to make the code smaller and easier to understand, while at the same time lowering the amount of time and effort needed to write a program.
+
+We've seen one of the common ways this is accomplished in programming, functions. While functions are occasionally used just to repeat the same code over and over, more often than not, they are executed with different input values. Structs, classes, and enums also allow us to reuse code involving both methods (functions) and properties (data), reusing the code in those structs, classes, and enums with many different values.
```
Comment (ErikSchierboom) [inline languages/swift/concepts/generic-functions/about.md:3]: The documents should be considered as stand-alone documents, so I would suggest to remove the "we've" seen bit.

File: languages/swift/concepts/generic-functions/links.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (ErikSchierboom) [inline languages/swift/concepts/generic-functions/links.json:1]: Maybe include the https://docs.swift.org/swift-book/LanguageGuide/Generics.html#ID181 link here?

File: languages/swift/exercises/concept/get-organized/.docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+You are an internationally renowned organizing guru; your books and lectures have helped people around the world to organize and declutter their lives. To do this you have developed a number of tools to help organize all sorts of things. You have a tool to organize `Int`s a tool to organize `String`s a tool to organize `Double`s and on and on. There's only one problem… you have so many tools, you can't keep them organized.
```
Comment (ErikSchierboom) [inline languages/swift/exercises/concept/get-organized/.docs/instructions.md:1]:
```suggestion
You are an internationally renowned organizing guru; your books and lectures have helped people around the world to organize and declutter their lives. To do this you have developed a number of tools to help organize all sorts of things. You have a tool to organize `Int`s, a tool to organize `String`s, a tool to organize `Double`s and on and on. There's only one problem… you have so many tools, you can't keep them organized.
```

This theme is brilliant! 🤣

File: languages/swift/exercises/concept/get-organized/.docs/instructions.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+What you need to do in order to declutter your life is write a single tool that can organize _any_ type of thing. You have two tasks to help you do just that.
+
+##1. Swap two elements in any array
```
Comment (ErikSchierboom) [inline languages/swift/exercises/concept/get-organized/.docs/instructions.md:5]:
```suggestion
## 1. Swap two elements in any array
```

File: languages/swift/exercises/concept/get-organized/.docs/instructions.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+For this task, you will need to write a function, `swapAt(_:_:_:)` where the first parameter is an inout array of elements of any type and the second and third parameters are `Int` indices of two elements in the array. If the two indices are valid indices, the function will swap the values at the two indices, returning the modified array via the inout parameter. If either index is invalid, the array should remain unchanged.
+
+##2. Bubble sort any array
```
Comment (ErikSchierboom) [inline languages/swift/exercises/concept/get-organized/.docs/instructions.md:9]:
```suggestion
## 2. Bubble sort any array
```

File: languages/swift/exercises/concept/get-organized/.meta/config.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+  "authors": [
+    {
+      "github_username": "wneumann",
+      "exercism_username": "wneumann"
+    }
+  ],
+  "contributors": [],
+  "forked_from": []
```
Comment (ErikSchierboom) [inline languages/swift/exercises/concept/get-organized/.meta/config.json:9]: These can be omitted.

File: languages/swift/exercises/concept/get-organized/.docs/instructions.md (hunk 5 above)
Comment (ErikSchierboom) [reply languages/swift/exercises/concept/get-organized/.docs/instructions.md:9]: I'm not entirely sure about the bubble sorting. I _think_ we discussed somewhere that we didn't want concept exercises to implement algorithms (practice exercises are perfect for that). Is that correct @iHiD?

Comment (ErikSchierboom) [reply languages/swift/exercises/concept/get-organized/.docs/instructions.md:9]: I'll wait with the rest of the review when @iHiD replied to the above comment.

Comment (iHiD) [reply languages/swift/exercises/concept/get-organized/.docs/instructions.md:9]: If we want to use a Bubble Sort, we need to be a little more explicit about how to solve it (ie, give the algorithm away). The reason for this is that we want Concept Exercises to be solvable by someone familiar in a language within 5mins, and having to work out algorithms can extend that. Algorithms can also feel like a real turnoff to lots of people (myself included), and intimidating to people who feel insecure as they don't have CS degrees, so it's important to emphasise that they're not being tested on the algorithm itself, just the using Swift (the fun bit!).

> For this task, you are you going to implement a [Bubble Sort][bubble-sort] in Swift. This a simple algorithm that repeatedly steps through a list, comparing adjacent elements and swapping them if they are in the wrong order, until the list is sorted.
> 
> You will need to write a function `bubbleSort(_:swapIf:)`. The first parameter of `bubbleSort` is an inout array of elements of any type. The second parameter is a function that takes two values of the same type as the elements of the array and returns `true` if the first value should appear _after_ the second element in the array.  The function should use the `swapIf` parameter to determine if two elements need to be swapped and `swapAt` from you first task to swap the elements when needed.

Sidenote: Is the function expecting the user to write the loop to do this? That wasn't clear to me and we should be explicit about it if so. The easiest way is to give example of the input and output.e.g.

```swift
bubbleSort([1,3,2,4])
=> [1,2,3,4]
```

**Comment by ErikSchierboom** [comment]
This Friday (Jan 29th 2021), we're going to explode this repo back into the individual track repositories and archive this repo.

As such, this PR needs to be merged by Friday 9am UTC, or it will need to be manually reopened against the track repo.

**Pull request closed without merging**